src/pages/AppConfigure.vue

```diff
@@ -26,6 +26,9 @@ const currentTab: Ref<ConfigureTabType> = ref('general')
 const router = useRouter()
 const store = useStore()
 const toast = useToast()
+const route = useRoute()
+
+currentTab.value = String(route.name).replace('Settings', '').toLowerCase()
```

Comment:
no need to cast route name, because it is already a string

Reply:
Need to cast it to string because `route.name` has a type of `RouteRecordName` and I get type error. Even if we don't add string it still works, but we keep getting those type errors. Added `String` to bypass those type errors

![Screenshot from 2022-06-17 10-14-19](https://user-images.githubusercontent.com/24249988/174226765-b061d69b-1eeb-4372-a1c6-31e411e6ad30.png)